includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -1249,7 +1249,7 @@ public function get_number_store_year( $table_name ) {
 		if( $table_exists ) {
 			// get year for the last row
 			$year = $wpdb->get_var( "SELECT YEAR(date) FROM {$table_name} ORDER BY id DESC LIMIT 1" );
-			// default to currenty year if no results
+			// default to currently year if no results
```

Comment:
should this be  'default to **current** year..' otherwise, all good

Reply:
Done 🍏